charts/drupal/values.yaml

```diff
@@ -797,9 +797,3 @@ signalsciences:
     limits:
       cpu: 40m
       memory: 300Mi
-
-# Logging configurations for services that support them.
-# BETA feature, currently only supported by drupal-cron.
-logging:
-  # Currently supported formats are "default" and "json".
-  format: default
```

Comment:
As we discussed do not remove this key from values.yaml or the schema. Deployments that have it will fail if you do, see https://app.circleci.com/pipelines/github/wunderio/drupal-project-k8s/5448/workflows/22d0526d-0c52-4643-a105-f907b3535a47/jobs/17480

Let's do the removal in a separate PR with proper deprecation notices etc etc